### PR TITLE
Use properties for costumes

### PR DESCRIPTION
**Pull request by Tymewalk**
So we don't have to use `set` functions.

Expands on #34.

### CODE REVIEW

**Comment by Tymewalk** [comment]
The first commit is just so I can create the pull request - I'm working on the new costume methods now.

**Comment by Tymewalk** [comment]
OK, I've made `costumeNumber` a property.
@BookOwl, I've never tried that before, I just made it based on your code. Is that right?

File: slither/__init__.py [hunk 1]
```diff
@@ -69,7 +70,17 @@ def costumeNumber(self):
...
+    @property
+    def costumeName(self):
+        '''The name of the costume the sprite is showing'''
+        return self._costumeName
+
+    @costumeName.setter
+    def costumeNumber(self, val):
+        self.setCostumeByName(val)
+
+    # TODO: Fix these
```
Comment (Tymewalk) [inline slither/__init__.py]: So I think that what has to be done here is all the checks and stuff done in `setCostumeByNumber/Name` go into `recalculateCostumeNumber/Name` functions.
Seems '_simple_'.

**Comment by BookOwl** [comment]
When do you think this will be ready?

**Comment by Tymewalk** [comment]
@BookOwl I completely forgot about this. There's not that much left to do, though, so it shouldn't be too hard.
Then we can release 0.2.0 (#44).

**Comment by Tymewalk** [comment]
Strange... my local Slither repo doesn't have this branch?

**Comment by BookOwl** [comment]
> Strange... my local Slither repo doesn't have this branch?

That's weird.

**Comment by BookOwl** [comment]
It's not in mine either. :/

**Comment by Tymewalk** [comment]
OK... I'll just try rewriting `__init__.py` to remove the conflicts. Then I'll just push to `newCostumeMethod`

**Comment by Tymewalk** [comment]
AGH WHY DOES MERGING ALWAYS MESS UP

**Comment by Tymewalk** [comment]
Anyways, the conflicts look to be fixed.

**Comment by BookOwl** [comment]
Great!

**Comment by Tymewalk** [comment]
Alright @BookOwl, how does it look?

**Comment by BookOwl** [comment]
@Tymewalk looks good to me!

**Comment by Tymewalk** [comment]
Don't merge it - I found another problem.

**Comment by BookOwl** [comment]
@Tymewalk ok, I'll let you merge when you finish

**Comment by Tymewalk** [comment]
Done. There's no more errors. I'm merging now and then we can get to work on #44.

**Comment by BookOwl** [comment]
Deleting newCostumeMethod

**Pull request merged**